layouts/shortcodes/classer.html

```diff
@@ -0,0 +1,3 @@
+<div class="{{ .Get "class" }}">
```

Comment:
this completely defeats the purpose of having shortcodes at all, it's almost the same as having custom HTML because you can put whatever still in there.

Every single custom thing should be put in its own self-contained shortcode